docs/docs/pages/style-spec/root.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Root
+id: root
+description: Root level properties of a MapLibre style specify the map's layers, tile sources and other resources, and default values for the initial camera position when not specified elsewhere.
+contentType: specification
+order: 2
+layout: page
+hideFeedback: true
+products:
+- MapLibre Style Specification
+prependJs:
+    - "import Items from '../../components/style-spec/items';"
+    - "import ref from '../../../../tsc/src/reference/latest';"
+---
+
+Root level properties of a MapLibre style specify the map's layers, tile sources and other resources, and default values for the initial camera position when not specified elsewhere.
+
+```json
+{
+    "version": {{ref.$version}},
+    "name": "Mapbox Streets",
+    "sprite": "mapbox://sprites/mapbox/streets-v{{ref.$version}}",
```

Comment:
We should probably change that to `https://...` as we don't support mapbox protocol anymore.